website/main.css

```diff
@@ -75,3 +75,26 @@ main > h2:first-child {
   border-top: 0;
   padding-top: 0;
 }
+
+@media (prefers-color-scheme: dark) {
+  a:link {
+    color: #3398e0;
+  }
+  a:visited {
+    color: #979ef8;
+  }
+  body {
+    background-color: #002b36;
+    color: white;
+  }
+  .os-logo[src="macos.svg"] {
+    filter: invert(100%);
+  }
+}
+
+@media (prefers-color-scheme: light) {
+  body {
+    background: #fff;
+    color: #333;
```

Comment:
Minor: Why are we changing the text color of light mode? Making it explicitly black is fine, but you're making it off-black here.

Reply:
Have made text explicitly black and background explicitly white. Have left prefers-color -scheme light in-case you want to change colors later on.